rosbot_description/urdf/components/orbbec_astra.gazebo.xacro

```diff
@@ -49,41 +49,43 @@
     </joint>
     <link name="orbbec_astra_depth_reference_link"></link>
 
-    <gazebo reference="orbbec_astra_depth_reference_link">
-      <sensor type="depth" name="orbbec_astra_camera">
-        <always_on>1</always_on>
-        <update_rate>10</update_rate>
+    <xacro:unless value="${simulation_engine == 'webots'}">
```

Comment:
I think that it will be better to use `if` and check if simulation_engine is gazebo